exporter/otlp/lib/opentelemetry/exporter/otlp.rb

```diff
@@ -4,8 +4,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-require 'opentelemetry/exporter/otlp/exporter'
-require 'opentelemetry/exporter/otlp/version'
+require "opentelemetry/exporter/otlp/version"
+require "opentelemetry/exporter/otlp/exporter"
```

Comment:
Rubocop is complaining 
```suggestion
require 'opentelemetry/exporter/otlp/version'
require 'opentelemetry/exporter/otlp/exporter'
```

Reply:
No worries, looks like CI is failing on something unrelated right now though 😞